### PR TITLE
Move to src directory

### PR DESCRIPTION
**Pull request by meatballs**
This PR will:

* Create a `src` directory
* Move the tests directory out of the source directory and up to root level
* Remove the pytest options from `setup.cfg` and set them explicitly in `.travis.yml` and `appveyor.yml`

Running tests locally will now mean using `python setup.py test` rather than `python -m pytest`
It also means that pep8 checking will no longer be performed locally and will have to be run separately if required.

### CODE REVIEW

**Comment by meatballs** [comment]
To set options for running tests locally, it's possible to create a `pytest.ini` file with whatever options are desired. In order that doing so doesn't then affect travis and appveyor, I'm going to add that file to .gitignore

**Comment by drvinceknight** [comment]
> baca34b

:+1:

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    git clone git@github.com:your_name_here/ConferenceScheduler.git
+
+3. Create a branch for your work::
+
+    git checkout -b name-of-your-branch
+
+4. Commit your changes and push your branch to Github::
+
+    git add -A
```
Comment (drvinceknight) [inline CONTRIBUTING.rst]: Perhaps we don't need to go in to the details here? (I personally do not recommend `git add -A` and `git commit -m ...`.

Comment (meatballs) [inline CONTRIBUTING.rst]: Fairly hastily typed! Shall I just take out the code line?

**Comment by drvinceknight** [comment]
This looks great to me. I've made one comment about the `pip install requirements.txt` but I'm not sure I'm actually suggesting anything there or not. Merge if you're happy :) :+1: Nice job getting this setup!

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+Testing
+-------
+
+ConferenceScheduler includes a comprehensive test suite. Please ensure that you run all the tests before submitting a Pull Request.
+
+To setup your environment for testing:
+
+1. Install the necessary Python libraries::
+
+    pip install -r requirements.txt
```
Comment (meatballs) [inline CONTRIBUTING.rst]: True. Shouldn't need that command

Pytest is listed in setup.py, so no idea why it wouldn't get pulled

Comment (drvinceknight) [inline CONTRIBUTING.rst]: I've just tried locally and it didn't... I thought it would. We could just replace this line with `pip install pytest`?

Comment (meatballs) [inline CONTRIBUTING.rst]: You wouldn't get the pep8 plugin (also listed in setup.py), so I reckon it's safer to use requirements.txt

Comment (meatballs) [inline CONTRIBUTING.rst]: It seems that the `tests_require` setting only specifies what should be installed - it doesn't instruct setup.py to actually go and get them! 

http://stackoverflow.com/questions/4734292/specifying-where-to-install-tests-require-dependencies-of-a-distribute-setupto#7747140

Comment (drvinceknight) [inline CONTRIBUTING.rst]: > You wouldn't get the pep8 plugin (also listed in setup.py), so I reckon it's safer to use requirements.txt

Fine by me :+1:

**Pull request merged**